lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -556,7 +556,11 @@ def get(self, request, course_key):
                     users, course_key=course_key, collected_block_structure=course_data.collected_structure
                 ):
                     if not exc:
-                        entries.append(self._gradebook_entry(user, course, graded_subsections, course_grade))
+                        enrollment = CourseEnrollment._get_enrollment_state(user, course_key)[0]
```

Comment:
Shouldn't have to query for the enrollment again, since it was already queried in _paginate_users

Reply:
I am examining paged_enrollments in debugger. It doesn't seem to have any information on track. Am I missing something?